core-web/apps/dotcms-ui/src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.html

```diff
@@ -22,6 +22,7 @@
     <div class="field">
         <label class="form__label">{{ 'contenttypes.content.push_publish.filters' | dm }}: </label>
         <p-dropdown
```

Comment:
Did you try this? https://primefaces.org/primeng/autofocus or maybe the autofocus property of this component?

Reply:
nice find!